pkg/defaulting/cluster.go

```diff
@@ -139,6 +139,13 @@ func DefaultClusterSpec(ctx context.Context, spec *kubermaticv1.ClusterSpec, tem
 	// default cluster networking parameters
 	spec.ClusterNetwork = DefaultClusterNetwork(spec.ClusterNetwork, kubermaticv1.ProviderType(spec.Cloud.ProviderName), spec.ExposeStrategy)
 
+	// If KubeLB is enforced, enable it.
+	if datacenter.Spec.KubeLB != nil && datacenter.Spec.KubeLB.Enforced {
+		spec.KubeLB = &kubermaticv1.KubeLB{
+			Enabled: true,
+		}
```

Comment:
I foresee bugs when the KubeLB struct is extended with more fields other than `Enabled`. Then this code will wipe everything else the user has set in the Cluster object. Maybe it's better to already write this a bit more carefully.

Reply:
Thanks for this one, otherwise it would have been a rigorous debugging session for someone :D